examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/TwitterJSONTagToADM.scala

```diff
@@ -63,11 +64,11 @@ object TwitterJSONTagToADM {
     val thpool = Executors.newFixedThreadPool(threadNumber)
     implicit val ec: ExecutionContext = ExecutionContext.fromExecutor(thpool)
 
-    val buffer = new ArrayBuffer[String](threadNumber)
+    val buffer = new ArrayBuffer[String](bufferSize)
```

Comment:
Add comments to the codebase: "We use a buffer to store a batch of records for multiple threads to process."